platforms/hyperledger-besu/charts/node_validator/templates/deployment.yaml

```diff
@@ -184,8 +185,6 @@ spec:
             containerPort: {{ .Values.node.ports.p2p }}
             protocol: TCP
         env:
-          - name: BESU_GENESIS_DIR
```

Comment:
this will delete the genesis file when the pod is restarted. Is that what we want?

Reply:
In the docker image for v1.5.5, a user named besu has been introduced, which is not having sudo permissions and can only have rights to write in the /tmp folder. the value of the BESU_GENESIS_DIR is being picked up from mountPath which is being used at other places too. hence didn't want to touch it or edit it, which is the reason why hardcoded the genesis dir path.
The genesis file will not get deleted as it is being picked up from the vault when the pod starts.